src/components/DappBrowser/control-panel/ReanimatedPager.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { PanGestureHandler } from 'react-native-gesture-handler';
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+  interpolate,
+  SharedValue,
+  useAnimatedGestureHandler,
+  withTiming,
+} from 'react-native-reanimated';
+import { TIMING_CONFIGS } from '@/components/animations/animationConfigs';
+import { AlignVertical, alignVerticalToFlexAlign } from '@/design-system/layout/alignment';
+import { deviceUtils } from '@/utils';
+import { clamp } from '@/__swaps__/utils/swaps';
+
+const DEVICE_WIDTH = deviceUtils.dimensions.width;
+
+export const ReanimatedPager = React.memo(function ReanimatedPager({
+  children,
```

Comment:
usually `React.memo` doesn't work as expected on components with a `children` prop
```ts
<ReanimatedPager>someText</ReanimatedPager> // string or number  works fine
<ReanimatedPager>
  <p>someText</p> // but this breaks memoization as it's creating a new component something like `createElement('p', ...)`
</ReanimatedPager> 
```

Reply:
I think you're right but committed this by mistake, was an old file